feconf.py

```diff
@@ -530,6 +530,10 @@ def get_empty_ratings():
 # the existing storage models for UserStatsModel.
 DASHBOARD_STATS_DATETIME_STRING_FORMAT = '%Y-%m-%d'
 
+# We Generate Images for existing math rich text components in batches. This
+# gives the maximum size for a batch of Math SVGs in bytes.
+MAX_SIZE_OF_MATH_SVG_BATCH_BYTES = 31 * 1024 * 1024
```

Comment:
I don't know if it's wise to go so close to the upper limit. Give it a bit of breathing room, e.g. 10 MB or a max number of entities at a time. It's OK if the admin has to click the button a bit more, but you don't want network latency or the number of items to update to become a problem.

Reply:
For now this limit will only be only used for calculation of minimum number of batches required in the audit job. 
The actual limit can be set once we see the result of the job(i.e we have an idea of how many explorations require conversion).
Also the actual size will be significantly less than what we estimate in the one-off job. So for example if the audit job suggests that an exploration has 10 Mb of SVGs, the actual size will be around 6 or 7Mb.
Also we will be calculating the actual size of the Payload in the frontend and set a limit there anyway.